howdy/src/compare.py

```diff
@@ -193,9 +194,19 @@ def send_to_ui(type, message):
 
 # Fetch the max frame height
 max_height = config.getfloat("video", "max_height", fallback=0.0)
-# Get the height of the image
-height = video_capture.internal.get(cv2.CAP_PROP_FRAME_HEIGHT) or 1
 
+# Get screen orientation
+landscape = True
+dsp = display.Display()
+screen_width = dsp.screen().width_in_pixels
+screen_height = dsp.screen().height_in_pixels
+if screen_height > screen_width:
```

Comment:
I'm worried this will trigger for people using external vertical monitors, i really think this should be disabled by config option by default